src/transformation/transformers/class/members/accessors.ts

```diff
@@ -0,0 +1,91 @@
+import * as ts from "typescript";
+import * as tstl from "../../../../LuaAST";
+import { TransformationContext } from "../../../context";
+import { createSelfIdentifier } from "../../../utils/lua-ast";
+import { transformFunctionBody, transformParameters } from "../../function";
+import { transformIdentifier } from "../../identifier";
+import { findInClassOrAncestor, isStaticNode } from "../utils";
+
+export function hasSetAccessorInClassOrAncestor(
+    context: TransformationContext,
+    classDeclaration: ts.ClassLikeDeclarationBase,
+    isStatic: boolean
+): boolean {
+    return (
+        findInClassOrAncestor(context, classDeclaration, c =>
```

Comment:
Don't know if we can do anything about this, but this formatting looks strange. Maybe extract the callback to its own const?